src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -0,0 +1,108 @@
+
+import meshio
+from meshio._mesh import CellBlock
+import numpy as np
+import argparse
+
+
+def convert_abaqus_to_gmsh(input_mesh, output_mesh):
+  """
+  @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information
+  @param input_mesh path of the input abaqus file
+  @param output_mesh path of the output gmsh file
+  """
+  # Load the mesh
+  print('Reading abaqus mesh...')
+  mesh = meshio.Mesh.read(input_mesh, "abaqus")
+
+  # Add a default region tag
+  print('Converting region tags...')
+  region_list = list(mesh.cell_sets.keys())
+  Nregion = len(region_list)
+  cell_ids = np.zeros(len(mesh.cells[0][1]), dtype=int)
+  for ii, ka in zip(range(Nregion), region_list):
+    mesh.field_data[ka] = [ii + 1, 3]
+    region_list.append(ka)
+    cell_ids[mesh.cell_sets[ka][0]] = ii + 1
+  mesh.cell_data['gmsh:physical'] = [cell_ids]
```

Comment:
Does `[cell_ids]` transform it into a 2-dim array? (use `shape` member function?)